templates/home.tmpl

```diff
@@ -1,158 +1,72 @@
-          </span>
-          <span class="stargazers">
-            <i class="fa fa-clock-o" title="Last updated"></i>
-            ${pkg["update_age"].str}
-          </span>
-        </div>
-        <div class="panel-footer">
-          <a class="btn input-group-btn" href="${pkg["html_url"].str}">
-            <i class="fa fa-globe"></i> Project website
-          </a>
-        </div>
-      </div>
+    <p class="w-message"><span>Package maintainer?</span> <a href="/about.html">Learn</a> what this service can do for you.</p>
+    </div>
     </div>
-    # end
-</div>
+  </div>
 
+  <div class="container medium" id="showcase">
 
-<h4><i class="fa fa-fire"></i>New packages</h4>
-<div class="columns">
-    # for p in new_packages:
-    <div class="column col-6">
-      <div class="panel package">
-        <div class="panel-header">
-          <div class="panel-title">
-            <a href="/pkg/${p["name"].str}">${p["name"].str}</a>
-            <!-- TODO: Latest version -->
-            <!-- <span class="version">v0.1.1</span> -->
-          </div>
-          <div class="package-description">
-            <p>${p["description"].str}</p>
-          </div>
-          # if p.hasKey("web"):
-          <a class="btn input-group-btn" href="${p["web"].str}">
-            <i class="fa fa-globe"></i> Project website
-          </a>
-          # end
-        </div>
-        <div class="panel-body">
-          <!-- TODO: The info below -->
-          <!-- <figure class="avatar avatar-xs person" data-initial="DP">
-            <img src="img/avatar-1.png" />
-          </figure>
-          <span class="username">dom96</span>
-          <span class="stargazers"><i class="fa fa-star" title="Stargazers"></i>456</span>
-          <span class="stargazers"><i class="fa fa-clock-o" title="Last updated"></i>2 minutes ago</span> -->
-        </div>
-        <div class="panel-footer">
-          <!-- TODO: Links for these buttons. -->
-          <!-- <a class="btn input-group-btn">
-            <i class="fa fa-github"></i> Repo
-          </a>
-          <a class="btn input-group-btn">
-            <i class="fa fa-book"></i> Docs
-          </a> -->
-        </div>
+    <div class="twelve columns" id="package-showcase">
+      <h2><i class="fas fa-poll" style="color:#2ff36b"></i> Trending on GitHub</h2>
+      # for i in 0 .. min(8, len(github_trending))-1:
+      # let pkg = github_trending[i]
+      <div class="four columns" id="pack-show">
+        <h4>${pkg["name"].str}</h4>
+        <span class="stars"><i class="fas fa-star" style="color:#f7e942"></i> ${pkg["stargazers_count"].getNum}</span>
+        <p class="show-text"><img src="${pkg["owner"]["avatar_url"].str}""><span class="author"><a href="${pkg["owner"]["html_url"].str}">${pkg["owner"]["login"].str}</a></span>
+          </p>
+        <p>${pkg["update_age"].str}</p>
+        <button class="visit-pack-btn"><a href="/pkg/${pkg["name"].str}/"><i class="fas fa-link"> </i> Visit</a></button>
```

Comment:
A "visit" button is uncommon. The package name should be clickable - perhaps with a hint about it.

Reply:
I put the link in the title. Do you want to remove the button totally?